robocop/utils/misc.py

```diff
@@ -378,3 +378,39 @@ def _is_var_scope_local(node) -> bool:
         if "scope=" in option.value:
             is_local = option.value.lower() == "scope=local"
     return is_local
+
+
+def parse_order_comma_sep_list(value: str, mapping: dict) -> list:
+    ordered = []
+    for item in value.split(","):
+        item_lower = item.lower()
+        if item_lower not in mapping:
+            raise ValueError(f"Invalid value: {item}. Supported values: {', '.join(mapping.keys())}")
```

Comment:
We are missing negative test scenario for this code. 

But I couldn't find such test easily in Robocop (I have usually done them in Robotidy). I will consider it as task for me in the future, leaving story here: https://github.com/MarketSquare/robotframework-robocop/issues/1091 (feel free to resolve this comment without doing anything)